ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -288,12 +288,15 @@ tests:
     workflow: cucushift-installer-rehearse-aws-ipi
 - as: olmv1-benchmark-test
   cluster: build05
-  cron: 0 0,4 * * *
+  cron: 0 0,4,8 * * *
   steps:
     allow_skip_on_success: true
     cluster_profile: gcp-qe
+    env:
+      NO_REPORTPORTAL: "true"
```

Comment:
Work around that RP issue, more: https://redhat-internal.slack.com/archives/CH76YSYSC/p1754451439933309?thread_ts=1754385612.115479&cid=CH76YSYSC